packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -101,260 +116,141 @@ export const Default = {
-        }
-      },
-      constants: [
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        },
-        {
-          type: 'attribute',
-          name: 'open',
-          value: true
-        }
-      ],
-      args
-    });
-  }
+  name: 'Distance',
+  render: () => html`
+    <sd-dropdown open distance="8">
+      <div class="slot slot--border slot--text p-4 w-[300px]">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
 };
 
 /**
- * The offset of the panel along the trigger can be customized using the `skidding` attribute. This value is specified in pixels.
+ * Use the `skidding` attribute to offset the panel away from its trigger.
  */
 
 export const Skidding = {
+  name: 'Skidding',
   parameters: { controls: { exclude: ['placement'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'placement',
-          values: ['bottom', 'right']
-        },
-        x: {
-          type: 'attribute',
-          name: 'skidding',
-          values: ['0', '-20', '20']
-        }
-      },
-      args
-    });
-  }
+  render: () => html`
+    <sd-dropdown open skidding="20" placement="bottom">
+      <div class="slot slot--border slot--text p-4 w-[300px]">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
 };
 
 /**
- * Use the `default` slot to place the content of the dropdown.
- * Use the `trigger` slot to place a trigger like e. g. a button.
+ * Use the `no-auto-size` attribute to prevent the dropdown from automatically adjusting its size.
  */
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['size'] }
-  },
-  decorators: [
-    (story: any) =>
-      html`<style>
-          .slot-highlight {
-            outline: purple 1px dashed;
-          }</style
-        >${story()}`
-  ],
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'slot',
-          name: 'default',
-          title: 'slot=...',
-          values: [
-            {
-              value: `<div class="slot slot--border slot--background example"></div><sd-button slot="trigger">Trigger</sd-button>`,
-              title: 'default'
-            },
-            {
-              value: `<div class="slot slot--text example"></div><sd-button class="slot slot--border slot--background" slot="trigger">Trigger</sd-button><div class="slot-overlay"  slot="trigger"></div>`,
-              title: 'trigger'
-            }
-          ]
-        }
-      },
-      constants: [{ type: 'slot', name: 'trigger', value: '' }],
-      args
-    });
-  }
+
+export const NoAutoSize = {
+  name: 'No Auto Size',
+  render: () => html`
+    <sd-dropdown no-auto-size open>
+      <div class="slot slot--border slot--background" style="height: 110vh; width: 120px;">Scroll down</div>
```

Comment:
In my opinion, "scroll down" should be at the start:
without scrolling:
![image](https://github.com/user-attachments/assets/3bc521f1-f100-42f3-8d05-96b716a98853)

Reply:
Updated